circle.yml

```diff
@@ -376,7 +389,48 @@ jobs:
           path: platform/android/MapboxGLAndroidSDKTestApp/lint-baseline.xml
 
 # ------------------------------------------------------------------------------
-  android-release-all:
+  android-build-release:
+    docker:
+      - image: mbgl/feb0443038:android-ndk-r17
+    resource_class: large
+    working_directory: /src
+    environment:
+      LIBSYSCONFCPUS: 4
+      JOBS: 4
+      BUILDTYPE: Release
+      IS_LOCAL_DEVELOPMENT: false
+    steps:
+      - checkout
+      - *generate-cache-key
+      - *restore-cache
+      - *restore-gradle-cache
+      - *reset-ccache-stats
+      - run:
+          name: Generate Maven credentials
+          shell: /bin/bash -euo pipefail
+          command: |
```

Comment:
If this exact command is run multiple times or in multiple jobs, you might consider extracting it out into a macro (or whatever Circle calls these steps here that start with `*`).

Reply:
Good point @friedbunny, was thinking about it as well. Here is the ticket https://github.com/mapbox/mapbox-gl-native/issues/12073.